assignment1/kthInLinkedList_test.py

```diff
@@ -0,0 +1,26 @@
+"""Tests for kthInLinkedList()."""
+
+from kthInLinkedList import *
+import unittest
+
+class kthInLinkedListTest(unittest.TestCase):
+    def setUp(self):
+        self.n = 15
+        self.head = Node(self.n - 1)
```

Comment:
I don't sure how to make private functions in the Python. As i know, `_` in the beginning of a function sname works smth like this. But I don't sure, may be it can be made by smth else.